ANSWERS.md

```diff
@@ -0,0 +1,8 @@
+## Questions
+1. What is the DOM?
+DOM stands for Document Object Model. It's basically a way to edit elements in a website (using JavaScript to access and edit HTML and CSS).
+If you like a challenge and don't want to make your life easier by using a jQuery library, the DOM is for you.
```

Comment:
Good job.  Keep in mind that jQuery's main focus is to perform efficient DOM manipulation, but it's still using the DOM and doing the same things as vanilla JS under the hood.